backend/docs/backend/api-docs/schedules.md

```
@@ -0,0 +1,394 @@
+
+---
+
+## POST /bandspaces/{bandspaceId}/schedules
+
+**설명:** 밴드 공간에 합주 일정을 생성한다.
+**인증:** 필요 (JWT Bearer)
+
+### Request
+
+**Path Parameters**
+| 파라미터 | 타입 | 필수 | 설명 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)